ref_builder/cli.py

```diff
@@ -214,6 +214,17 @@ def otu_promote_accessions(
     type=str,
     required=True,
 )
+@click.option(
+    "--ignore-name",
+    is_flag=True,
+    default=False,
+    help="ignore isolate names in Genbank sources",
+)
+@click.option(
+    "--isolate-name",
+    type=(IsolateNameType, str),
+    help='an overriding name for the isolate, e.g. "isolate ARWV1"',
```

Comment:
Does this actually work? The string gets parsed into a tuple?

Reply:
Yes, but [here's the doc](https://click.palletsprojects.com/en/8.1.x/options/#tuples-as-multi-value-options) also

Reply:
Wow!